standup/script.js

```diff
@@ -219,10 +219,15 @@ async function searchButtonHandler() {
   const usernames = searchInput.value
     .split(',')
     .map((username) => username.trim());
-  const filteredUsernames = [...new Set(usernames)];
+  const formattedUsernames = usernames.map((username) => `user:${username}`);
```

Comment:
will the current implementation allow duplicate usernames?

Reply:
No